tests/main_pipeline_alevin.test

```diff
@@ -0,0 +1,68 @@
+nextflow_pipeline {
+
+    name "Test Workflow main.nf"
+    script "main.nf"
+
+    test("test-dataset_alevin_aligner") {
+
+        when {
+            // the rest is taken from shared config
+            params {
+                aligner    = 'alevin'
+                outdir     = "${outputDir}/results_alevin"
+
+                // Limit resources so that this can run on GitHub Actions -- for some reason it had not been taken from shared config
+                max_cpus   = 2
+                max_memory = '6.GB'
+                max_time   = '6.h'
+            }
+        }
+
+        then {
+
+            //
+            // General assertions
+            //
+
+                // Did it finish successfully?
+                assert workflow.success
+
+                // How many tasks were executed?
+                assert workflow.trace.tasks().size() == 16
+
+                // How many results were produced?
+                assert path("${outputDir}/results_alevin").list().size() == 5
+                assert path("${outputDir}/results_alevin/alevin").list().size() == 4
+                assert path("${outputDir}/results_alevin/alevin/mtx_conversions").list().size() == 4
+                assert path("${outputDir}/results_alevin/alevinqc").list().size() == 2
+                assert path("${outputDir}/results_alevin/fastqc").list().size() == 12
+                assert path("${outputDir}/results_alevin/multiqc").list().size() == 3
+
+            //
+            // Check if files were produced
+            //
+            assert new File( "${outputDir}/results_alevin/alevin/Sample_X_alevin_results/af_quant/alevin/quants_mat_cols.txt" ).exists()
+            assert new File( "${outputDir}/results_alevin/alevin/Sample_X_alevin_results/af_quant/alevin/quants_mat.mtx" ).exists()
+            assert new File( "${outputDir}/results_alevin/alevin/Sample_X_alevin_results/af_quant/alevin/quants_mat_rows.txt" ).exists()
+
+            assert new File( "${outputDir}/results_alevin/alevin/Sample_Y_alevin_results/af_quant/alevin/quants_mat_cols.txt" ).exists()
+            assert new File( "${outputDir}/results_alevin/alevin/Sample_Y_alevin_results/af_quant/alevin/quants_mat.mtx" ).exists()
+            assert new File( "${outputDir}/results_alevin/alevin/Sample_Y_alevin_results/af_quant/alevin/quants_mat_rows.txt" ).exists()
```

Comment:
Do we need this if the same files are already part of the snapshot?

Reply:
Nope. Not needed.
This is a "footprint" from the conversion which I forgot to remove. I will take it out from all the tests where it happens.

Reply:
Just fixed this one.